reinforcement_learning/bindings/cs/rl.net.native/rl.net.config.cc

```diff
@@ -0,0 +1,19 @@
+#include "rl.net.config.h"
+
+API u::configuration* CreateConfig()
```

Comment:
Is it possible to have a better name instead of `u` here?

Reply:
Ah, whoops, I forgot to yank the shorthand